packages/atomic/src/components/search/result-template-components/result-template-decorators.spec.tsx

```diff
@@ -65,8 +68,8 @@ describe('resultContext method', () => {
       template: () => (
         <atomic-result
           content={document.createElement('div')}
-          engine={TestUtils.buildMockSearchAppEngine()}
           result={TestUtils.buildMockResult()}
+          interactiveResult={null as unknown as InteractiveResult}
```

Comment:
Just to confirm, we wouldn't actually do this "in real life", right? If this is actually a valid use-case for the `interactiveResult` prop, I feel like it may be best to just make said prop optional. Moreover, if this _**is**_ something we want to support outside of tests, there should be a UT to confirm the `atomic-result` component renders as expected when said prop is missing ✅

Reply:
So, this test suite _does not_ test the atomic-result. This test suite test the `@ResultContext` decorator, which has nothing to do with this prop.

I could have put a ts-ignore and it would have been all the same.

I don't know we want to allow a result to be rendered without interactiveResult 🤔 
What I know is that this is not a change that should be made in this PR 😄